packages/opds-web-client/src/AuthPlugin.ts

```diff
@@ -1,6 +1,9 @@
 import { AuthFormProps, AuthButtonProps } from "./components/AuthProviderSelectionForm";
 import { AuthMethod, AuthCredentials } from "./interfaces";
 
+/** Applications can implement this interface if they would like to support authentication
+    methods other than basic auth. A list of `AuthPlugin`s can be passed as a prop to the
```

Comment:
Don't you need to start new lines with `*` and start the comment after the first line? Unless if this gets read the same way?

Reply:
I haven't actually tried running this through typedoc, but it's a valid comment. I'm not sure.